src/main/java/com/aws/greengrass/clientdevices/auth/configuration/RuntimeConfiguration.java

```diff
@@ -52,6 +61,9 @@ public final class RuntimeConfiguration {
     static final String CERTS_V1_KEY = "v1";
     static final String CERTS_STATUS_KEY = "s";
     static final String CERTS_STATUS_UPDATED_KEY = "l";
+    private static final String HOST_ADDRESSES_KEY = "hostAddresses";
+    private static final UpdateBehaviorTree HOST_ADDRESSES_UPDATE_BEHAVIOR =
+            new UpdateBehaviorTree(UpdateBehaviorTree.UpdateBehavior.REPLACE, System.currentTimeMillis());
```

Comment:
you can't store this. You need to always use the current system time.

Reply:
whoops fixed